ch01data/084Boids.ipynb

```diff
@@ -4,7 +4,9 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "## The Boids!"
+    "## The Boids!\n",
+    "\n",
+    "This section shows an example of using NumPy to encode a model of how a group of birds or other animals moves. It is based on [a paper](http://www.cs.toronto.edu/~dt/siggraph97-course/cwr87/) by Craig W. Reynolds. Reynolds calls the simulated creatures \"bird-oids\" or \"boids\", so that's what we'll be calling them here too."
```

Comment:
```suggestion
    "This section shows an example of using NumPy to encode a model of how a group of birds or other animals moves. It is based on [a paper by Craig W. Reynolds](http://www.cs.toronto.edu/~dt/siggraph97-course/cwr87/). Reynolds calls the simulated creatures \"bird-oids\" or \"boids\", so that's what we'll be calling them here too."
```